docs/source/notebooks/wps_generate_climos.ipynb

```diff
@@ -0,0 +1,388 @@
+    "    dry_run=dry_run\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[<metaurl mediatype=\"text/plain\">\n",
+      "     http://localhost:5001/outputs/5690ae31-0368-11eb-b7e1-c9fa1b0c3983/fdd_seasonal_CanESM2_rcp85_r1i1p1_1951-2100_dry.txt\n",
+      "    </metaurl>, <metaurl mediatype=\"text/plain\">\n",
+      "     http://localhost:5001/outputs/5690ae32-0368-11eb-b7e1-c9fa1b0c3983/gdd_annual_CanESM2_rcp85_r1i1p1_1951-2100_dry.txt\n",
+      "    </metaurl>]\n"
+     ]
+    }
+   ],
+   "source": [
+    "req = requests.get(dry_output.get()[0])\n",
+    "soup = BeautifulSoup(BeautifulSoup(req.content.decode('utf-8')).prettify()).find_all('metaurl')\n",
+    "print(soup)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Test dry output for multiple files\n",
+    "assert len(soup) == 2\n",
+    "    \n",
+    "url1 = soup[0].get_text()\n",
+    "with urlopen(url1) as f:\n",
+    "    output_data1 = f.read().decode('utf-8')\n",
+    "    \n",
+    "assert re.search(r'time_resolution: seasonal', output_data1)\n",
+    "    \n",
+    "url2 = soup[1].get_text()\n",
+    "with urlopen(url2) as f:\n",
+    "    output_data2 = f.read().decode('utf-8')\n",
+    "    \n",
+    "assert re.search(r'time_resolution: yearly', output_data2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Process normal output for multiple files\n",
+    "\n",
+    "output = thunderbird.generate_climos(\n",
+    "    netcdf=[seasonal_opendap,annual_local], \n",
+    "    operation=operation, \n",
+    "    climo=climo, \n",
+    "    resolutions=resolutions, \n",
+    "    dry_run=False\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[<metaurl mediatype=\"application/x-netcdf\">\n",
+      "     http://localhost:5001/outputs/5f9f9acc-0368-11eb-b7e1-c9fa1b0c3983/gdd_aClimMean_CanESM2_rcp85_r1i1p1_19610101-19901231.nc\n",
+      "    </metaurl>, <metaurl mediatype=\"application/x-netcdf\">\n",
+      "     http://localhost:5001/outputs/5f9f9acd-0368-11eb-b7e1-c9fa1b0c3983/fdd_aClimMean_BCCAQ_CanESM2_historical+rcp85_r1i1p1_19610101-19901231_Canada.nc\n",
+      "    </metaurl>]\n"
+     ]
+    }
+   ],
+   "source": [
+    "req = requests.get(output.get()[0])\n",
+    "soup = BeautifulSoup(BeautifulSoup(req.content.decode('utf-8')).prettify()).find_all('metaurl')\n",
+    "print(soup)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Test multiple files\n",
+    "assert len(soup) == 2"
```

Comment:
Great! Did you consider adding another check here for the data itself?

Reply:
Yes I did, but as I mentioned I wasn't sure what to do. I will go off of what you said for the single file output.